cmsplugin_articles_ai/management/commands/publish_test_articles.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from cmsplugin_articles_ai.factories import TaggedArticleFactory
```

Comment:
Made the factory boy optional and included factories and management command into the installation.